docs/contrib/04_custom_implementations.md

```diff
@@ -1,6 +1,61 @@
 # Custom implementation guidelines
 
-... in construction ...
+`pySDC` solves (non-)linear ODE of the form
+
+$$
+\frac{du}{dt} = f(u, t), \quad u(0)=u_0, \quad t \in [0, T].
+$$
+
+where $u$ a vector or scalar, representing one or more variables.
+
+The type of variable, the definition of the right-hand side $f(u,t)$ and initial solution value $u_0$ are defined in a given `Problem` class.
+
+... to be continued ...
```

Comment:
What's supposed to go here?

Reply:
The full definition of a problem class (with base methods, abstract ones, etc ...). Supposed to come later following #253